src/main/java/webdriver/util/TestListener.java

```diff
@@ -0,0 +1,73 @@
+package webdriver.util;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.testng.ITestContext;
+import org.testng.ITestListener;
+import org.testng.ITestResult;
+import webdriver.driver.DriverManager;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class TestListener implements ITestListener {
+    private Logger log = LogManager.getRootLogger();
+
+    @Override
+    public void onTestStart(ITestResult iTestResult) {
+
```

Comment:
Not to leave the methods empty you can log some information inside: iTestResult.getTestName() + " Test has started."